packages/core/src/render3/features/external_styles_feature.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentDefFeature} from '../interfaces/definition';
+
+/**
+ * A feature that adds support for external runtime styles for a component.
+ * An external runtime style is a URL to a CSS stylesheet that contains the styles
+ * for a given component. For browsers, this URL will be used in an appended `link` element
+ * when the component is rendered.
+ *
+ * @codeGenApi
+ */
+export function ɵɵExternalStylesFeature(styleUrls: string[]): ComponentDefFeature {
+  return (definition) => {
```

Comment:
nit (more for readability vs typing): 
```suggestion
  return (definition: ComponentDef<unknown>) => {
```